src/app/speechtotext/speechtotext.component.ts

```diff
@@ -86,7 +86,8 @@ export class SpeechtotextComponent implements OnInit {
 
       if (t === 5) {
         this.subscription.unsubscribe();
-        this.store.dispatch(new speechactions.SearchAction(false));
+        this.message = "Please check your microphone and audio levels.";
+        // this.store.dispatch(new speechactions.SearchAction(false));
```

Comment:
@nikhilrayaprolu but using `if (t==6)` condition after `if (t==4)` is not being executed. I tried dispatching action at `t==6` but it does not works.